packages/commitlint-config/test/ignore-rules.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, expect, test } from "vitest";
+import { ignoreDependabot } from "../src/rules/ignore-rules";
+
+describe("ignoreDependabot", () => {
+  test("should return true when the message includes `dependabot[bot]`", () => {
```

Comment:
nit: capitalize the test description. Just extrapolating from the casing rules for comments.

```diff
-"should return true when the message includes `dependabot[bot]`"
+"Should return true when the message includes `dependabot[bot]`"
```